tests/Geb/positive/app-fst-pair.geb

```diff
@@ -0,0 +1,15 @@
+(app
+    int 
+    int
```

Comment:
This should be (prod int int)?

Reply:
I wrote the example just for the evaluator. So, the types are not meant to be correct.